man/systemd.network.xml

```diff
@@ -612,6 +612,17 @@
         </para></listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>IPv6ProxyNDP=</varname></term>
+          <listitem><para>A boolean. Configures proxy NDP. Proxy NDP is a
```

Comment:
I think it would be good to briefly mention here that NDP means Neighbor Discovery Protocol. I.e. the first time this is mentioned, add `(Neighbor Discover Protocol)` after it...